fecgov/fec-cms
# PAC & party datatable filter panel updates

## Summary (required)

- Resolves #4687 

Rearrange filters into accordions, removed authorized committee filters from panel

### Required reviewers

1 front end & 1 UX

## Impacted areas of the application

General components of the application that this PR will affect:

-  PAC and party datatable

## Screenshots

**Mockup**
![mockup](https://user-images.githubusercontent.com/31663028/122299236-b8db6100-cecb-11eb-8cb0-da8a2b23286e.jpg)
**Actual**
<img width="1425" alt="Screen Shot 2021-06-30 at 2 56 51 PM" src="https://user-images.githubusercontent.com/31420082/124016291-79c10b80-d9b3-11eb-929c-db145ae72520.png">


## Related PRs

Related PRs against other branches:

branch | PR
------ | ------
feature/4651-minimal-pac-party-datatable | [link](https://github.com/fecgov/fec-cms/pull/4653)
feature/4686-single-select-time-period | [link](https://github.com/fecgov/fec-cms/pull/4692)
feature/4688-pac-party-details-panel | [link](https://github.com/fecgov/fec-cms/pull/4709)

## How to test

(Include any information that may be helpful to the reviewer(s). This might include links to sample pages to test or any local environmental setup that is unusual such as environment variable (never credentials), API version to point to, etc)

- Check out this branch
- Wait for https://github.com/fecgov/openFEC/pull/4900 to be merged or check out openFEC branch `feature/4878-add-filters` and connect to the `dev` database. Follow "Steps to test" in that PR to run locally.
- Go to http://localhost:8000/data/committees/pac-party/
- Test the filters (except leadership PAC sponsor and SSFs - API needs org type added (see https://github.com/fecgov/openFEC/issues/4879)

### Future work
https://github.com/fecgov/fec-cms/issues/4690 
https://github.com/fecgov/fec-cms/issues/4737
